model.py

```diff
@@ -5884,8 +5902,16 @@ def get(cls, _db, url, do_get=None, extra_request_headers=None,
             max_age = max_age.total_seconds()
 
         # Do we already have a usable representation?
+        #
+        # 'Usable' means we tried it and either got some data or
+        # received a status code outside the 2xx series.
         usable_representation = (
-            representation and not representation.fetch_exception)
+            representation and not representation.fetch_exception
+            and (
+                representation.content or representation.local_path
+                or representation.status_code / 100 != 2
```

Comment:
`!= 2` is what I meant, but on reflection I think it's wrong.

If the status code is in the 200 range then we get some .content. I'm trying to capture the fact that even status codes outside the 200 range can be cached. 

For example when we look up an ISBN in OCLC+LD we get a 302 redirect. I'd like to reuse that redirect next time instead of making the HTTP request again.

Some OCLC+LD files give a 404 error. I'd like to cache the 404 error on the same terms as I would cache a successful response.

This is what `test_302_creates_cachable_representation` tests.

I think what I actually want here is to verify that the status code was not in the 5xx range, because that's the only time you might reasonable expect that making the request again would give you a different result.